test/views/behaviours/SummarySubscriptionComponentBehaviours.scala

```diff
@@ -33,7 +33,7 @@ trait SummarySubscriptionComponentBehaviours extends ViewBehaviours {
         val npsData: Map[Int, Int] = userAnswers.get(NpsData)(NpsDataFormats.formats).get
         val taxYears: Seq[TaxYearSelection] = userAnswers.get(SummarySubscriptionsPage)(PSubsByYear.formats)
           .get.map(year => getTaxYearPeriod(year._1)).toSeq
-        val applyView = view.apply(subscriptions, npsData, navigator.nextPage(SummarySubscriptionsPage, NormalMode, userAnswers).url, NormalMode)(fakeRequest, messages)
+        val applyView = view.apply(subscriptions, npsData, navigator.nextPage(SummarySubscriptionsPage, NormalMode, userAnswers).url, NormalMode, true)(fakeRequest, messages)
```

Comment:
Should this be a parameter that is passed into the behaviour? Otherwise, we would never be able to test the false case 🤔?

Reply:
Agreed, I have moved the tests to within this component as it already tests both sides (true and false)